intelmq/bots/experts/abusix/lib.py

```diff
@@ -22,7 +22,7 @@ def query(ip):
         query = rev + QUERY_HOST
 
         try:
-            response = dns.resolver.query(query, 'TXT')
+            response = dns.resolver.resolve(query, 'TXT')
```

Comment:
That method is new in dnspython 2.0.0 (see the changelog https://dnspython.readthedocs.io/en/stable/whatsnew.html). I'm therefore not sure if we should break compatibility with the old versions.

Reply:
Yeah, you're right. I've removed that commit.

Reply:
Once the compatibility layer gets removed, we'd need to support both dnspython 1.x and >= 2.x by using different function names, or a compat layer.